.flake8

```diff
@@ -1,3 +1,5 @@
 [flake8]
 max-line-length=200
-
+ignore = E, C, W, D
```

Comment:
Do we still need the flake8 file with `ruff`?

Reply:
Thanks @jtpio for catching it!